### PR TITLE
MongoDb VS CassandraDB: the 2's consideration

### LINKED ISSUE #5828: MONGO DB VS CASSANDRA DB: THE 2’S CONSIDERATION
#### Proposed article introduction
Database management also known as DBM, refers to the actions a business takes to manipulate and control data needed to meet the necessary conditions throughout the entire data life-cycle. In practice, numerous tools have been created and utilized for the purpose of managing different forms and volumes of data with one of the most widely used being Mongo DB and Cassandra DB. 

In this article, we'll take you on a journey of database management using Mongo DB and Cassandra DB with the end goal of providing sufficient information to future users on the suitability of these two tools particularly in the area of Speed and Security (2’S).
#### Key takeaways 
This article aims at providing information regarding:
- Mongo DB and Cassandra DB. 
- Similarities and Positives that exists in Mongo DB and Cassandra DB. 
- Evaluation of in-use difference between Mongo DB and Cassandra DB particularly on the basis of Database Security and Speed.
#### Article quality
This article provides an in-depth view of two of the most in vogue data management systems.

#### References
N/A

### Conclusion
At the end of the tutorial, the reader should be able to achieve the goals listed above and on further learning, improve their database management knowledge and usability.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Bankanthony**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5828 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by Bankanthony** [comment]
@hectorkambow this article has been open for 13 days and no one has touched it. What's the issue

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

File: content/authors/Femi-ige-Mobolaji/index.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Femi-ige Mobolaji is a growing tech enthusiast trying to connect with a growing community. He is an entrepreneurship major from the Federal University of Minna.
```
Comment (Jethro-magaji) [inline content/authors/Femi-ige-Mobolaji/index.md:1]: Please edit your author bio in the proper Markdown format. Here's an [example](https://raw.githubusercontent.com/section-engineering-education/engineering-education/master/content/authors/adith-bharadwaj/index.md)

Also, add your avatar i.e (author image).

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]
Hi! @auxesis 
Well done for such an amazing article. However, you need to add more stuffs/content in your article to be able to make a good comparison between the 2 technologies. Security and speed isn't good enough for the comparison could you add something on; pricing, code structure, usage etc. 

Here's a template [guide](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md) for you.

Please, run your article on tools like Grammarly and ProWritingAid for better editing. This would save the review time to review your work.
Thanks.

**Pull request closed without merging**